viewer/packages/utilities/src/WebGLRendererStateHelper.test.ts

```diff
@@ -118,22 +212,60 @@ describe('WebGLRendererStateHelper', () => {
     const setScissorTestSpy = jest.spyOn(renderer, 'setScissorTest');
 
     helper.setScissorTest(false);
-    expect(setScissorTestSpy).toBeCalledWith(false);
+    expect(setScissorTestSpy).toHaveBeenCalledWith(false);
 
     helper.resetState();
-    expect(setScissorTestSpy).toBeCalledWith(true);
+    expect(setScissorTestSpy).toHaveBeenCalledWith(true);
   });
 
   test('setScissor()', () => {
+    const renderer = mockedWebGLRenderer().object();
+
     renderer.setScissor(0, 0, 64, 64);
     const helper = new WebGLRendererStateHelper(renderer);
     const setScissorSpy = jest.spyOn(renderer, 'setScissor');
 
     helper.setScissor(0, 0, 128, 128);
-    expect(setScissorSpy).toBeCalledWith(0, 0, 128, 128);
+    expect(setScissorSpy).toHaveBeenCalledWith(0, 0, 128, 128);
 
     helper.resetState();
-    expect(setScissorSpy).toBeCalledWith(new THREE.Vector4(0, 0, 64, 64));
+    expect(setScissorSpy).toHaveBeenCalledWith(new THREE.Vector4(0, 0, 64, 64));
+
+    function mockedWebGLRenderer(): Mock<THREE.WebGLRenderer> {
```

Comment:
Is it necessary with separate renderer-mocking functions in this file? Is it because this test is testing very specific parts of the renderer?